JeroenArsenal/JNA/fn_vehicleArsenal.sqf

```diff
@@ -676,7 +677,8 @@ switch _mode do {
 					jnva_loadout set [_index,[jnva_loadout select _index,[_item,_count]] call jn_fnc_arsenal_removeFromArray];
 					_mass = ["getMassItem",[_item,_count,_index]] call jn_fnc_vehicleArsenal;
 					jnva_loadout_mass = jnva_loadout_mass - _mass;
-					[_index, _item, _count] remoteExecCall ["jn_fnc_arsenal_addItem"];
+					//[_index, _item, _count] remoteExecCall ["jn_fnc_arsenal_addItem"];
+					[_index, _item, _count] call jn_fnc_arsenal_addItem; //Sparker: why execute it on all clients?
```

Comment:
This line and also the line 660 above are done by sparker, it avoid execute on all clients and causing the strange number values